### PR TITLE
Binary Classification Razor Pages Model Builder Tutorial

### PR DESCRIPTION
**Pull request by luisquintanilla**
[AB#1573477](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1573477)

[Review Link](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis-model-builder?branch=pr-en-us-13796)

To-Do:

- [x] Link code snippets to working sample. Pending PR dotnet/machinelearning-samples#646

### CODE REVIEW

File: docs/machine-learning/tutorials/sentiment-analysis-model-builder.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+
+### Create sentiment analysis handler
+
+Predictions will be made inside the main page of the application. Therefore, a method that takes the user input and uses the `PredictionEnginePool` to return a prediction needs to be added.
+
+1. Open the `Index.cshtml.cs` file located in the `Pages` directory and add the following using statements:
+
+```csharp
+using Microsoft.Extensions.ML;
+using SampleModelBuilderWebAppML.Model.DataModels;
```
Comment (natke) [inline docs/machine-learning/tutorials/sentiment-analysis-model-builder.md]: Correct web app name ...

File: docs/machine-learning/tutorials/sentiment-analysis-model-builder.md [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+
+    Finally, return the percentage back to the web page.
+
+    ```csharp
+    return Content(percentage.ToString("0.0"));
+    ```
+
+### Configure the web page
+
+The results returned by the `OnGetAnalyzeSentiment` will be dynamically displayed on the `Index` web page.
```
Comment (natke) [inline docs/machine-learning/tutorials/sentiment-analysis-model-builder.md]: Number the instructions in this section like the previous ones

File: docs/machine-learning/tutorials/sentiment-analysis-model-builder.md [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+1. Create a global variable to store the location of the trained model file.
+
+```csharp
+private readonly string _modelPath;
+```
+
+1. The model file is stored in the build directory alongside the assembly files of your application. To make it easier to access, create a helper method called `GetAbsolutePath` after the `Configure` method
+
+```csharp
+public static string GetAbsolutePath(string relativePath)
```
Comment (natke) [inline docs/machine-learning/tutorials/sentiment-analysis-model-builder.md]: I had to include System.IO for this to work;

**Comment by natke** [comment]
I haven't got the application working yet. Will do some more debugging and let you know how it goes.

**Comment by luisquintanilla** [comment]
For `Index.cshtml`, add entire page including `@page` and `@model` attributes to avoid confusion.

**Comment by luisquintanilla** [comment]
Triggerint re-build

**Comment by luisquintanilla** [comment]
@briacht @rustd @natke Ready for review.

File: docs/machine-learning/tutorials/sentiment-analysis-model-builder.md [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+
+    1. Finally, return the sentiment back to the web page.
+
+        [!code-csharp [ReturnSentiment](~/machinelearning-samples/samples/modelbuilder/BinaryClassification_Sentiment_Razor/SentimentRazor/Pages/Index.cshtml.cs#L32)]
+
+### Configure the web page
+
+The results returned by the `OnGetAnalyzeSentiment` will be dynamically displayed on the `Index` web page.
+
+1. Open the *Index.cshtml* file in the *Pages* directory and replace its content with the following code:
```
Comment (natke) [inline docs/machine-learning/tutorials/sentiment-analysis-model-builder.md]: Should this be `contents`?

**Review by natke — APPROVED** [review]
One minor comment. Otherwise, looks great!

File: docs/machine-learning/tutorials/sentiment-analysis-model-builder.md (hunk 4 above)
Comment (luisquintanilla) [reply docs/machine-learning/tutorials/sentiment-analysis-model-builder.md]: yep. Good catch.

**Pull request merged**